modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -182,6 +184,58 @@ public ObjectValidationRule updateObjectValidationRule(
 		return objectValidationRulePersistence.update(objectValidationRule);
 	}
 
+	@Override
+	public void validate(
+			Object object, long objectDefinitionId,
+			Map<String, Object> modelAttributes)
+		throws PortalException {
+
+		if (object == null) {
+			return;
+		}
+
+		HashMapBuilder.HashMapWrapper<String, Object> hashMapWrapper =
+			HashMapBuilder.<String, Object>putAll(
+				modelAttributes
+			).put(
+				"currentUserId",
+				() -> {
+					if (PrincipalThreadLocal.getUserId() > 0) {
+						return PrincipalThreadLocal.getUserId();
+					}
+
+					return null;
+				}
+			);
+
+		List<ObjectValidationRule> objectValidationRules =
```

Comment:
We need to get the SystemObjectFields in the Validations tab and not the CustomObjectFields